jdaviz/app.py

```diff
@@ -1995,7 +2024,11 @@ def vue_data_item_remove(self, event):
 
         data_label = event['item_name']
         data = self.data_collection[data_label]
-        self._reparent_subsets(data)
+        if self.config == "imviz":
```

Comment:
Also have to check link type here? You don't want this happening when linked by pixel, right? Or is the link type being handled somewhere else?

Reply:
Can we check something else other than the config?  Maybe check if the orientation plugin is loaded (which could probably be done in one line to avoid building helper.plugins multiple times, or by accessing the tray_items which is probably cheaper)?